packages/r/retroshare/package.yml

```diff
@@ -24,8 +25,17 @@ builddeps  :
     - pkgconfig(sqlcipher)
     - pkgconfig(x11)
     - pkgconfig(xscrnsaver)
+environment: |
+    export CC=clang
+    export CXX=clang++
```

Comment:
Will it work if we do `clang: yes` here instead of exporting the environment variables?